src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -124,17 +124,22 @@ public void setPerson(Person target, Person editedPerson) {
         return filteredPersons;
     }
 
-    @Override
-    public ObservableList<Person> getViewedPersonList() {
-        return viewedPerson;
-    }
-
     @Override
     public void updateFilteredPersonList(Predicate<Person> predicate) {
         requireNonNull(predicate);
         filteredPersons.setPredicate(predicate);
     }
 
+    //=========== Filtered Person List Accessors =============================================================
+
+    /**
+     * Returns an unmodifiable view of the list of currently viewed person.
+     */
+    @Override
+    public ObservableList<Person> getViewedPersonList() {
```

Comment:
This is for the schedule card right? I think the method name should change to something better since it is not technically a person list. Maybe `getScheduleView()`?

Reply:
I changed it to getViewSchedulePerson() cos I think the Person should be there.